admin/client/src/state/forms/README.md

```diff
@@ -0,0 +1,24 @@
+# forms
```

Comment:
We should make a call if we want reducer names (and state keys) to be singular or plural. We have `records` in plural, but `campaign` and `schema` in singular. I think since we can't rely on the value of this key to be a list/array in all cases (sometimes it'll be an object), it's more consistent to use singular.